### DISCUSSION TITLE
Time to cut a new release?

### DISCUSSION THREAD

**Discussion by hkjels** [wojtekmaj/react-date-picker#547] [General]
Vite complains that `Default and named imports from CSS files are deprecated`. I can see that this has been resolved on `main`, so I think it's worth a patch release. Is it trivial to publish a new version?

**Comment by wojtekmaj** [comment]
Hi,
the latest releases so not import CSS files directly, so this problem does not occur. If it does, let me know, provide reproduction and I'll look into it.

**Comment by hkjels** [comment]
I can see that there's been a release since I created this ticket, so this is not a problem any more
